### PR TITLE
Advanced template support in Intellij Idea

### PR DESCRIPTION
**Pull request by patricklx**
## Ember Times review template

- [x] Add your name to top and bottom
- [x] Put emoji in writeup title
- [x] Add a short blurb (could be the title) to the beginning
- [x] Link to external article/repo/etc in paragraph/body text, not just the writeup title (link in writeup title is optional now)
- [x] Add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account
- [x] Check that all links work

### CODE REVIEW

File: content/the-ember-times-issue-201.md [hunk 1]
```diff
@@ -1,28 +1,35 @@
...
 ---
 
-## [🐹 1. Section title in sentence case](section-url)
+## [💻 1. Advanced template support in Intellij Idea](https://github.com/patricklx/intellij-emberjs-experimental)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+In Intellij Idea the support for Emberjs templates has been quite lacking, Especially compared to vscode (Un)Stable Ember Language Server. With [intellij-emberjs-experimental](https://github.com/patricklx/intellij-emberjs-experimental) I added many features to the original plugin to improve the UX of working/coding with templates. Search for the addon named `Ember Experimental.js` and give it a try. **You will need to disable or remove the other Ember.js plugin to prevent conflicts.**
```
Comment (MinThaMie) [inline content/the-ember-times-issue-201.md]: Please replace `I` with your name and Github handle so people know it was you :)

File: content/the-ember-times-issue-201.md [hunk 2]
```diff
@@ -1,28 +1,35 @@
...
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+This plugin adds:
+* **Smart** Handlebars references for tags/mustache paths and tag attributes. By smart i mean that in handles ember helpers such as `let`, `hash`, `or`, `each`, `yield`. References to javascript/typescript and its able to resolve array item type in `each`.
```
Comment (MinThaMie) [inline content/the-ember-times-issue-201.md]: Maybe rephrase the `By smart I mean` to something like: This means that ....

**Review by MinThaMie — COMMENTED** [review]
Thanks for your write-up :) It definitively helps when the author of the "add-on" writes it. You could improve a bit on the way it's written to make it more descriptive instead of from an I-stand point if that makes sense. 

Feel free to ping me here or on Discord!

**Review by jaredgalanis — COMMENTED** [review]
thank you for your contribution to the Times! 🎉
